metadata-ingestion/src/datahub/cli/delete_cli.py

```diff
@@ -265,6 +265,11 @@ def undo_by_filter(
     type=str,
     help="Urn of the entity to delete, for single entity deletion",
 )
+@click.option(
+    "--urn-file",
+    required=False,
+    help="Absolute path of file with urns (one per line) to be deleted",
```

Comment:
Is being an absolute path required?

```suggestion
    help="Path of file with urns (one per line) to be deleted",
```